src/components/userProfile/UserProfile.js

```diff
@@ -52,6 +53,18 @@ class UserProfile extends Component {
       let currentPosition = this.state.currentPosition;
       let currentLocation = this.state.currentLocation;
 
+      let sectionIDs = ['Experience', 'Education'];
+      let i;
+      let j;
```

Comment:
What are these two variables for?
I can't find a usage yet. 
It this is used at somewhere, I think it would be good if the name is clear.